codalab/apps/web/static/js/worksheet/worksheet_side_panel.js

```diff
@@ -328,7 +329,7 @@ function renderDependencies(bundle_info) {
   </div>);
 }
 
-function renderMetadata(bundle_info) {
+function renderMetadata(bundle_info, updateContent) {
```

Comment:
Add a note that right now, we are updating the entire worksheet, but in the long term we want to just update the relevant worksheet item.  Let's call it refreshWorksheet to make it clear what this callback actually does.  'updateContent' is too vague.  Or it might be better to call it bundleMetadataChanged, and pass in the bundle info (current implementation is just to ignore it and call refreshWorksheet, but later we can do more)